host/python/uhd/usrp/multi_usrp.py

```diff
@@ -81,6 +81,7 @@ def _start_stream(streamer):
             if not stream_cmd.stream_now:
                 if start_time is not None:
                     stream_cmd.time_spec = start_time
+                    stream_cmd.stream_now = False
```

Comment:
This line is not necesssary (see the if clause 3 lines above, it's a requirement for coming here in the first place).